src/behat/UtilsContext.php

```diff
@@ -357,43 +357,100 @@ public function emptySelectTwo($cssId)
      */
     public function selectToSelectTwo($cssId, $what)
     {
-        // Open select2.
-        $this->assertFind('css', $cssId)->getParent()->find('css', 'span.select2-selection')->click();
-
         $this->spin(
-            function ($context) {
-                return $context->assertFind('css', '.select2-container--open .select2-search__field')->isVisible();
+            function ($context) use ($what, $cssId) {
+                $context->getSession()->evaluateScript(
+                    <<<JS
+                        (function() {
+                            let \$search = $('{$cssId}').data('select2').dropdown.\$search || $('{$cssId}').data('select2').selection.\$search;
```

Comment:
Is it necessary to prefix this javascript variable with the **$** symbol?

Reply:
yes, otherwise it throws an exception for undefined variable

Reply:
Not mandatory, but it is a jquery element and convention seems to be with `$` prefix